TrackingBackend/app/processes/osc.py

```diff
@@ -1,14 +1,18 @@
-from __future__ import annotations
+from app.utils import WorkerProcess, OneEuroFilter
 from app.config import EyeTrackConfig, OSCConfig
-from app.utils import WorkerProcess
 from app.types import EyeData, TrackerPosition
 from app.logger import get_logger
-from queue import Queue
+from queue import Queue, Empty
+from copy import deepcopy
+import numpy as np
 import threading
+import cv2
 from pythonosc.dispatcher import Dispatcher
 from pythonosc.udp_client import SimpleUDPClient
 from pythonosc.osc_server import ThreadingOSCUDPServer
 
+WIDTH = 256
+HEIGHT = 256
 
```

Comment:
In this case i dont think this is necessary since it is only used when creating the OSC debug window and isnt re-used anywhere else in the code.
In the future once more algorithms have been added i do think a constans.py file will benefit us